docs/source/international_languages.md

```diff
@@ -0,0 +1,232 @@
+   as an example.
+
+   Customize the `README.md` file for your new language.
+
+1. Add a `.readthedocs.yml` file [like this
+   one](https://github.com/hyperledger-labs/fabric-docs-ml/blob/master/.readthedocs.yml)
+   to the top level folder.  This file is configured to disable ReadTheDocs PDF
+   builds which may fail if you use non-latin character sets. Your top level
+   repository folder will now look like this:
+
+   ```bash
+   (base) anthonyodowd/git/fabric-docs-sa ls -al
+   total 96
+   ...
+   -rw-r--r--   1 anthonyodowd  staff    574  5 Jun 15:49 .readthedocs.yml
+   -rw-r--r--   1 anthonyodowd  staff  11358  5 Jun 14:38 LICENSE
+   -rw-r--r--   1 anthonyodowd  staff   4822  5 Jun 15:09 README.md
+   drwxr-xr-x  11 anthonyodowd  staff    352  5 Jun 14:38 docs
+   ```
+
+1. Commit these changes locally to your local repository:
+
+   ```bash
+   git add .
+   git commit -s -m "Initial commit"
+   ```
+
+1. Create a new repository on your GitHub account called `fabric-docs-sa`. In
+   the description field type `Hyperledger Fabric documentation in Sanskrit
+   language`.
+
+1. Update your local git `origin` to point to this repository, replacing
+   `YOURGITHUBID` with your GitHub ID:
+
+   ```bash
+   git remote set-url origin git@github.com:YOURGITHUBID/fabric-docs-sa.git
+   ```
+
+1. Push your `release-2.1` branch to be the `master` branch in this repository:
+
+   ```bash
+   git push origin release-2.1:master
+
+   Enumerating objects: 6, done.
+   Counting objects: 100% (6/6), done.
+   Delta compression using up to 8 threads
+   Compressing objects: 100% (4/4), done.
+   Writing objects: 100% (4/4), 1.72 KiB | 1.72 MiB/s, done.
+   Total 4 (delta 1), reused 0 (delta 0)
+   remote: Resolving deltas: 100% (1/1), completed with 1 local object.
+   To github.com:ODOWDAIBM/fabric-docs-sa.git
+   b3b9389be..7d627aeb0  release-2.1 -> master
+   ```
+
+1. Verify that your new repository `fabric-docs-sa` is correctly
+   populated on GitHub under the `master` branch.
+
+1. Connect your repository to ReadTheDocs using these
+   [instructions](./docs_guide.html#building-on-github). Verify that your
+   documentation builds correctly.
+
+1. When you are happy with your repository, create a request to make an
+   equivalent repository in the Hyperledger Labs organization, following [these
+   instructions](https://github.com/hyperledger-labs/hyperledger-labs.github.io).
+
+   Here's an [example
+   PR](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/126)
+   to show you the process at work.
+
+## Get connected
+
+Here's a few ways to connected with other people interested in international
+language translations:
+
+  * Rocket chat
+
+    Read the conversation or ask a question on the [Fabric documentation rocket
+    channel](https://chat.hyperledger.org/channel/fabric-documentation). You'll
+    find beginners and experts sharing information on the documentation.
```

Comment:
There is actually an internationalization channel that we started on Rocket Chat. That might be a better one to share here?
https://chat.hyperledger.org/channel/i18n

Reply:
or maybe we share both channels - general questions about the documentation content itself would go to https://chat.hyperledger.org/channel/fabric-documentation and questions about the translation process or other translations would go to https://chat.hyperledger.org/channel/i18n ?

Reply:
I think share both channels! thx for suggestion!